### ISSUE TITLE
Support for angular>1.4

### ISSUE THREAD

**Issue by vrt2015** [darthwade/angular-loading#17]
Hi,

Can you please provide support for angular versions >1.4

**Comment by WuglyakBolgoink** [comment]
+1

1) please fix directive name:
now work with:    "darthwade.dwLoading"

2) for angular >1.4 spin.js must write before angular

//= ../../bower_components/spin.js/spin.min.js
//= ../../bower_components/angular/angular.min.js

**Comment by jerrybk108** [comment]
Now I must open this library and change darthwade.loading to darthwade.dwLoading, but I can not use $loading.start($key); to start loading and $loading.finish($key); to stop loading; beside that I can not use bower to install this library.

**Comment by mattheworres** [comment]
The latest version is not recognized by Bower through the version numbers - 0.1.4 is the newest you can pull in via Bower. You can instead do this to install via the commandline:

`bower install angular-loading#e30ad76`

or add the following in your project's bower.json:

`"angular-loading": "e30ad76"`

I successfully use this library in Angular 1.5.x

**Comment by jerrybk108** [comment]
Hi mattheworres,

Thanks for your help!
When I added your version, then I installed this package into my project, I got the issue: "ECONFLICT Unable to find suitable version for angular".
I think your bower has an issue with angular version requirement.

**Comment by mattheworres** [comment]
@jerrybk108 yes, due to the Bower.json of this project being out of date (as per the request of this issue, and a few PRs update it as well I think), Bower can't find a suitable version that matches the constraints, and thus will warn you.

If you resolve the Angular version to the version you're using, in most instances Bower will provide its warning, but will pull the libraries down as needed. As I said, I've been using angular-loading in Angular 1.5 without any issue, so it works. There may be a few edge cases that don't perform as expected, but I haven't found them.
